src/main/java/core/basesyntax/FruitShop.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import core.basesyntax.service.FileService;
+import core.basesyntax.service.FileServiceImpl;
+import core.basesyntax.service.ReportBuilder;
+import core.basesyntax.service.ReportBuilderImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class FruitShop {
+    public static final Map<String,Integer> FRUIT_WAREHOUSE = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<String,Integer> fruitWaehouse = new HashMap<>();
```
константи це примітиви або незмінні класи (String) , так як мапа в ході програми змінюється тут потрібно використати дефолтний неймінг